misc/config_tools/scenario_config/validator.py

```diff
@@ -101,7 +107,7 @@ def check_semantics(self, board_etree, scenario_etree):
 
     @staticmethod
     def format_paths(unified_node, parent_map, report_on, variables):
-        elems = elementpath.select(unified_node, report_on, variables = variables, parser = elementpath.XPath2Parser)
+        elems = elementpath.select(unified_node, report_on, variables=variables)
```

Comment:
Why do we want to drop the `parser` optional parameter?

Also I see quite a few stylish changes in this file.

Reply:
Maybe merge issue...Sorry for that..